### PR TITLE
Deepcopy GraphModule in minifier

### LINKED ISSUE #90397: [dynamo] Accuracy minifier doesn't seem to catch bug
### 🐛 Describe the bug

The following program doesn't appear to trigger the accuracy minifier.

As per the [troubleshooting docs](https://pytorch.org/docs/master/dynamo/troubleshooting.html#accuracy-debugging) I tried:
```
TORCHDYNAMO_REPRO_AFTER="dynamo" TORCHDYNAMO_REPRO_LEVEL=4 python /tmp/repro.py
TORCHDYNAMO_REPRO_AFTER="aot" TORCHDYNAMO_REPRO_LEVEL=4 python /tmp/repro.py
```

But neither seem to catch the bug.

```
from typing import List

import torch
import torch._dynamo as dynamo
import operator

def miscompile_div_as_mul_backend(gm: torch.fx.GraphModule,
                                  example_inputs: List[torch.Tensor]):
    for node in gm.graph.nodes:
        if node.op == "call_function":
            if node.target == operator.truediv:
                node.target = operator.mul
    gm.recompile()
    return gm

@dynamo.optimize(miscompile_div_as_mul_backend)
def f(x, y):
    a = x * y
    b = x + y
    c = x / y
    return a, b, c


args = (torch.tensor([1., 2., 3.]), torch.tensor([4., 5., 6.]))
print(f(*args))
```

### Versions

PyTorch version: 1.14.0.dev20221205+cpu
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Debian GNU/Linux rodete (x86_64)
GCC version: (Debian 12.2.0-3) 12.2.0
Clang version: 14.0.6-2
CMake version: version 3.25.0
Libc version: glibc-2.35

Python version: 3.10.8 (main, Nov  3 2022, 15:17:13) [GCC 12.2.0] (64-bit runtime)
Python platform: Linux-5.19.11-1rodete1-amd64-x86_64-with-glibc2.35
Is CUDA available: False
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.24.0rc2
[pip3] torch==1.14.0.dev20221205+cpu
[pip3] torchvision==0.15.0.dev20221205+cpu
[conda] Could not collect

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh @mlazos @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### PR DESCRIPTION
**Pull request by williamwen42**
Fixes https://github.com/pytorch/pytorch/issues/90397. Remove deepcopy calls in minifier tests.

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

**Review by anijain2305 — APPROVED** [review]

**Review by mlazos — APPROVED** [review]

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>

**Comment by clee2000** [comment]
### Merge started
Your change will be merged once all checks pass (ETA 0-4 Hours).

Learn more about merging in the [wiki](https://github.com/pytorch/pytorch/wiki/Bot-commands).

Questions? Feedback? Please reach out to the [PyTorch DevX Team](https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours)<details><summary>Advanced Debugging</summary>
Check the merge workflow status 
<a href="None">here</a>
</details>
